### PR TITLE
Auth-Api docs

### PR DESCRIPTION
**Pull request by sampler36**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please add a relevant label.

## Change status

- [x] Complete, tested, ready to review and merge

- [x] Complete, but not tested (may need new tests)

- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project

- [x] I have performed a self-review of my own code

- [x] I have commented my code, particularly in hard-to-understand areas

- [ ] I have made corresponding changes to the documentation, if relevant

- [ ] I have run the app using my feature and ensured that no functionality is broken

- [ ] My changes generate no new warnings

- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by sampler36** [comment]
Resolved #39 
Resolved #37

File: routes/endpoint-docs/authDocs.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ *  @apiVersion 0.1.0
+ *  @apiName postAuth
+ *  @apiGroup Auth
+ *
+ *  @apiParam {json} user information
+ *  @apiParamExample {json} Request-Example:
+ *    {
+ *      "name": "nicolas group",
+ *      " password": "1234",
+ *        "email": "nicolasgroup@gmail.com"
```
Comment (Pav0l) [inline routes/endpoint-docs/authDocs.js:12]: please add to request example:
```
"hotel_id": "5cc74ab1f16ec37bc8cc4cdb",
"motto": "Cross-platform executive application",
 "user_type": "receptionist"
```

File: routes/endpoint-docs/authDocs.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ *
+ *  @apiSuccessExample Success-Response: registered
+ *    HTTP/1.1 200 OK
+ *    {
+ *       "_id":userId"5cc9839d0b92251d9ab4a1f4",
+ *      "name":"Merle",
+ *      "email": "nicolasgroup@gmail.com",
+ *     "token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoibmljb2xhcyBncm91cCIsImlhdCI6MTU1NjcxMDMwMX0.RdXMwTg3AMjiquiacVVMWL_kyUTOgdjTZLTdrPxzfoA"
+ *    }
+ *
```
Comment (Pav0l) [inline routes/endpoint-docs/authDocs.js:36]: Success response has this format:
```
{
    "user": {
        "_id": "5cc9b28740daa6367c5d0283",
        "hotel_id": "5cc74ab1f16ec37bc8cc4cdb",
        "name": "blabla",
        "email": "aazzaa@hotmail.com",
        "motto": "Cross-platform executive application",
        "user_type": "receptionist",
        "__v": 0
    },
    "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiYmxhYmxhIiwiaG90ZWxfaWQiOiI1Y2M3NGFiMWYxNmVjMzdiYzhjYzRjZGIiLCJpYXQiOjE1NTY3MjIzMTF9.zI9RKAe7Gr8kfctrMH_51ixu2P0v6yNQXhPZfwo6m8I"
}
```

**Review by Pav0l — CHANGES_REQUESTED** [review]
Good job with the docs @sampler36 ! 🔥 

Had couple of comments below:

**Review by Pav0l — APPROVED** [review]

**Pull request merged**